search/processor.hpp

```diff
@@ -150,12 +135,21 @@ class Processor : public my::Cancellable
   void SetViewportByIndex(m2::RectD const & viewport, size_t idx, bool forceUpdate);
   void ClearCache(size_t ind);
 
+  /// @name Get ranking params.
+  //@{
+  /// @return Rect for viewport-distance calculation.
+  m2::RectD const & GetViewport(ViewportID vID = DEFAULT_V) const;
+  //@}
+
+  void SetLanguage(int id, int8_t lang);
```

Comment:
It would be nice to specify a type for id, and introduce an enum if there's no one.

Reply:
It's tricky, I'll do it in a followup PR.